src/main/java/unicash/ui/TransactionCard.java

```diff
@@ -43,8 +53,23 @@ public class TransactionCard extends UiPart<Region> {
     @FXML
     private Label transactionLocation;
 
+    //@FXML
+    //private Label categories;
+
+    @FXML
+    private Label firstCategory;
+
+    @FXML
+    private Label secondCategory;
+
+    @FXML
+    private Label thirdCategory;
+
     @FXML
-    private Label categories;
+    private Label fourthCategory;
+
+    @FXML
+    private Label fifthCategory;
```

Comment:
Is there a reason to hardcode each category to a label instead of defining them inside something like a FlowPane?

Reply:
Good idea, I actually didn't think of that I just went with the more obvious way.

Reply:
Ok after trying FlowPane, the styling changed q a bit and I'm not sure how to fix it, but in the mean time as I work on fixing that, and since the labels being tied to each category doesn't affect functionality nor the UI directly, I think that we can move on with just the Labels for now, and at a later iteration we can think about simplifying the code, after the more important major features and functionalities have been implemented.